lenovo/ideacentre/k330/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, ... }:
+{
+    imports = [
+        ../../../common/cpu/intel
+        ../../../common/gpu # Is it possible/advisable to pin this to the 390.xx driver family in case the user wants to use non-free drivers?
+        ../../../pc
+    ];
+
+    # On my machine Wayland causes the desktop to freeze after a short time of operation
+    services.xserver.enable = true;
```

Comment:
I think we should rather document this behavior i.e. a README rather than enabling xserver.